plugins/command-line/prism-command-line.js

```diff
@@ -4,6 +4,67 @@ if (typeof self === 'undefined' || !self.Prism || !self.document) {
 	return;
 }
 
+// Scoped to this IIFE so they can be accessed by both hooks. -- cwells
+var outputLines = [];
+var outputRanges = [];
+
+Prism.hooks.add('before-highlight', function (env) {
+	var pre = env.element.parentNode;
+	if (!pre || !/pre/i.test(pre.nodeName)) {
+		return;
+	}
+
```

Comment:
You have to do the test for the `command-line` class here as well.

You could move all the tests from the `complete`-hook here and add a flag to the `env` object, so you don't have to test it again for the other hooks.